packages/react-bloc/.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
```

Comment:
I think we should merge this with the `.travis.yml` in the root of the repo since that's what will be used by travis